controllers/operands/kubevirt.go

```diff
@@ -280,9 +280,11 @@ func hcWorkloadUpdateStrategyToKv(hcObject *hcov1beta1.HyperConvergedWorkloadUpd
 			*kvObject.BatchEvictionSize = *hcObject.BatchEvictionSize
 		}
 
-		if size := len(hcObject.WorkloadUpdateMethods); size > 0 {
+		filterWorkloadUpdateMethods := filterWorkloadupdatemethods(hcObject.WorkloadUpdateMethods)
+
+		if size := len(filterWorkloadUpdateMethods); size > 0 {
 			kvObject.WorkloadUpdateMethods = make([]kubevirtcorev1.WorkloadUpdateMethod, size)
-			for i, updateMethod := range hcObject.WorkloadUpdateMethods {
+			for i, updateMethod := range filterWorkloadUpdateMethods {
 				kvObject.WorkloadUpdateMethods[i] = kubevirtcorev1.WorkloadUpdateMethod(updateMethod)
 			}
 		}
```

Comment:
check if the `copy` builtin function can be used here.

Reply:
no because we also need a typecast between string and kubevirtcorev1.WorkloadUpdateMethod